### PR TITLE
Config for JUWELS Booster (JSC)

### PR DESCRIPTION
**Pull request by caropen**

### CODE REVIEW

File: configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: JUWELS-Booster
+  site: JSC
+  system: JUWELS-Booster-rome-A100-Infiniband
+  integrator:
+    vendor: tba
```
Comment (caropen) [inline configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml]: @AndiH, can you fill this?

**Comment by caropen** [comment]
@AndiH @jayeshbadwaik (from Jülich) please also check

File: configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml (hunk 1 above)
Comment (AndiH) [reply configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml]:
```suggestion
    vendor: Atos
```

File: configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    name: tba
```
Comment (AndiH) [inline configs/JSC-JUWELS-Booster-rome-A100-Infiniband/system_definition.yaml]:
```suggestion
    name: XH2000
```

**Review by AndiH — COMMENTED** [review]
Added my parts! The filename contains `rome`, should it – maybe – be `zen2`?

File: configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+compilers:
+- compiler:
+      spec: nvhpc@=23.7
+      paths:
+        cc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc
+        cxx: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc++
+        f77: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+        fc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+      operating_system: rocky9
```
Comment (SebastianAchilles) [inline configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml]:
```suggestion
      operating_system: rocky8
```

JUWELS Booster is using Rocky 8, c.f. 

```
[jwlogin21 ~]$ cat /etc/os-release | grep PRETTY_NAME
PRETTY_NAME="Rocky Linux 8.9 (Green Obsidian)"
```

File: configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      target: aarch64
```
Comment (SebastianAchilles) [inline configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml]:
```suggestion
      target: x86_64
```

JUWELS Bosoter is using AMD Zen2 which are x86_64.

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Review by pearce8 — APPROVED** [review]

**Pull request merged**